toponetx/classes/simplicial_complex.py

```diff
@@ -136,22 +147,35 @@ def shape(self) -> tuple[int, ...]:
         Returns
         -------
         tuple of ints
```

Comment:
Since we are following type hint, the proper syntax would be somthing like tuple[int, int] for a tuple of two ints or tuple[int, ...] for variable number of ints in the tuple. See https://docs.python.org/3/library/typing.html#annotating-tuples

Reply:
This is actually as per NumPy's recommendation. I know we mix the two right now, but neither is wrong. If we want to unify this, that should be a separate PR fixing it everywhere and not just here :)

Reply:
Yeah, resolving for now. I have a personal preference for type hint one.